### ISSUE TITLE
NetworkManager integration

### ISSUE THREAD

**Issue by vlaci** [vlaci/openconnect-sso#1]
There is an existing OpenConnect plugin for NetworkManager.

**Comment by nickurak** [comment]
Seems like this work is active in openconnect: https://gitlab.com/openconnect/openconnect/-/merge_requests/75

**Comment by eotfofiw** [comment]
> There is an existing OpenConnect plugin for NetworkManager.

Yes, but this is for openconnect only right, and doesn't allow the sso flow to work - correct me if I'm wrong. 

Work just brought in sso for Any Connect and this means I don't have to use the awful cisco client, so thank you very much for the work!

**Comment by TheToto** [comment]
It seems SSO is now working in upstream

**Comment by YuriySamorodov** [comment]
Seems like openconnect-sso is not there yet
![image](https://user-images.githubusercontent.com/5260172/180681521-8cb03c1e-3720-4f35-bf08-c364317b195f.png)

**Comment by SergeyChepurko** [comment]
I think for Gnome users it's not an issue anymore since openconnect now able to open SSO browser window in there. But for KDE users it's still an active issue and openconnect-sso is the only option to connect to VPN. @vlaci Plz consider this enhancement.

**Comment by rapgro** [comment]
In Fedora installed NetworkManager-openconnect-gnome to enable plugin for NetworkManager via Gtk4. But after input of PIN generated by OTP (Cisco Duo) I can not continue and VPN stays disconnected neither any button of Connect, Login nor Close gets any result to properly connect. - How to let NM detect success from WebEngine as in that dialog obviously used webbrowser addin?
